gateways/firstdata/request_builders.go

```diff
@@ -0,0 +1,62 @@
+package firstdata
+
+import (
+	"strconv"
+
+	"github.com/BoltApp/sleet"
+)
+
+func buildAuthRequest(authRequest *sleet.AuthorizationRequest) (*Request, error) {
+
+	amountStr := sleet.AmountToString(&authRequest.Amount)
+
+	request := &Request{
+		RequestType: "PaymentCardPreAuthTransaction",
+		TransactionAmount: TransactionAmount{
+			Total:    amountStr,
+			Currency: authRequest.Amount.Currency,
+		},
+		PaymentMethod: PaymentMethod{
+			PaymentCard: PaymentCard{
+				Number:       authRequest.CreditCard.Number,
+				SecurityCode: authRequest.CreditCard.CVV,
+				ExpiryDate: ExpiryDate{
+					Month: strconv.Itoa(authRequest.CreditCard.ExpirationMonth),
+					Year:  strconv.Itoa(authRequest.CreditCard.ExpirationYear)[2:],
```

Comment:
lets add comment explaining we only use 2 digit year? Maybe also make this a little more robust (like a separate function we can also unit test)

Reply:
Added the comment in the types file. I havent pushed the unit test code yet but it should be covered by the unit test for the requestBuilder. Can discuss again after that push